src/features/simd.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Enable High Performance Workloads using WebAssembly SIMD, and V8'
+author: 'Deepti Gandluri ([@dptig](https://twitter.com/dptig)), Thomas Lively ([@tlively52](https://twitter.com/tlively52))'
+date: 2019-17-12
+tags:
+  - WebAssembly
+description: 'Bringing vector operations to WebAssembly'
+---
+
+## Introduction
+
+SIMD stands for Single Instruction, Multiple Data. SIMD instructions are a special class of instructions that exploit data parallelism in applications by simultaneously performing the same operation on multiple data elements. Compute intensive applications like audio/video codecs, image processors, are all examples of applications that take advantage of SIMD instructions to accelerate performance. Most modern architectures support some variants of SIMD instructions. 
+
+The WebAssembly SIMD proposal defines a portable, performant subset of SIMD operations that are available across most modern architectures. This proposal derived many elements from the [SIMD.js proposal](https://github.com/tc39/ecmascript_simd), which in turn was originally derived from the [Dart SIMD](https://www.researchgate.net/publication/261959129_A_SIMD_programming_model_for_dart_javascriptand_other_dynamically_typed_scripting_languages) specification. The SIMD.js proposal was an API proposed at TC39 with new types and functions for performing SIMD computations, but this was archived in favor of supporting SIMD operations more transparently in WebAssembly. The [WebAssembly SIMD proposal](https://github.com/WebAssembly/simd) was introduced as away for browsers to take advantage of the data level parallelism using the underlying hardware.
+
+## WebAssembly SIMD Proposal
+
+The high level goal of the WebAssembly SIMD proposal is to introduce vector operations to the WebAssembly Specification, in a way that guarantees portable performance. 
+
+The set of SIMD instructions is large, and varied across architectures. The set of operations included in the WebAssembly SIMD proposal consist of operations that are well supported on a wide variety of platforms, and are proven to be performant. To this end, the current proposal is limited to standardizing Fixed-Width 128-bit SIMD operations. 
```

Comment:
```suggestion
The set of SIMD instructions is large, and varied across architectures. The set of operations included in the WebAssembly SIMD proposal consists of operations that are well supported on a wide variety of platforms, and are proven to be performant. To this end, the current proposal is limited to standardizing Fixed-Width 128-bit SIMD operations. 
```

Reply:
"consist" -> "consists"

Reply:
I think consist may still be the right word here because it's referring to the set of operations and not the proposal.

Reply:
Ack, but the set *consists* of stuff, because it's a single set, no?

Reply:
+1, consists seems correct here.

Reply:
@dtig Sounds about right. Let's accept the suggestion then?